### ISSUE TITLE
Placeholder Text - Color does not meet accessibility

### ISSUE THREAD

**Issue by vavalos5** [grommet/hpe-design-system#2567]
Sherry Mead had pointed out how our placeholder text does not meet accessibility. Upon doing a bit of research, text not only needs to be 14pts for bold text or 18pts for regular, it requires a minimum or 3:1 ratio to be considered sufficient. Currently, the ratio is 1:92 w/ 18pts (default).

**Comment by vavalos5** [comment]
I will adjust this ticket later. Here's a link I found helpful to conclude we're not meeting accessibility for placeholder text: https://www.maxability.co.in/2013/11/27/1-4-3-contrast-minimum/

**Comment by ericsoderberghp** [comment]
> Our placeholder text is no longer text-xweak but rather text-weak, so it is accessible now.
